### PR TITLE
master-lps-122860

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LRQA-54536
https://issues.liferay.com/browse/LPS-122861
https://issues.liferay.com/browse/LPS-122860

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/1008<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/78705a8b6cba~1...78705a8b6cba)

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/assetpublisher/APConfiguration.path [hunk 1]
```diff
@@ -260,6 +260,11 @@
 	<td>//div[contains(.,'Asset Entries')]/following-sibling::div//a[@title='Select']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ASSET_SELECTION_MANUAL_ASSET_ENTRIES_SELECT_IN_DEPOT</td>
+	<td>//div[contains(.,'Asset Entries')]/following-sibling::div//a[@title='Select in Test Depot Name']</td>
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/assetpublisher/APConfiguration.path:265]: @liuxingye this should be more flexible. I've moved the hardcoding to the macro level, but if we're going to be writing more tests using different AL names then we should change this at the macro level as well.

Comment (liuxingye) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/assetpublisher/APConfiguration.path:265]: yeah, noticed your changes, thanks @austinchiang

**Pull request closed without merging**